src/cli/load-config.ts

```diff
@@ -363,10 +364,11 @@ export async function loadConfigFile(errors: string[] = []): Promise<{normalized
 
   let raw: any = undefined;
   const staticPublishRcPath = path.resolve('./static-publish.rc.js');
+  const staticPublishRcFileURL = url.pathToFileURL(staticPublishRcPath).toString()
```

Comment:
toString() is solely there for TypeScript (https://github.com/microsoft/TypeScript/issues/42866#issuecomment-857298254)